README.md

```diff
@@ -151,6 +115,11 @@ side of the table, either!
 Negotiating with candidates and fielding questions from candidates and employees requires
 understanding the same complex technicalities of equity compensation well.
 
+That said, this topic is **not simple** and we ask that readers be willing to invest time to
+get through a lot of confusing detail.
+If you’re in a hurry, or you don’t care to learn the details, this Guide may not be for
```

Comment:
Like I said in INTERNAL, I'm not happy telling readers what they want or what we think we know about them. They could come for a quick answer, read a little, and think, "Oh, I do want details, this is interesting or helpful." We have to give them the chance to surprise themselves. Selecting readers is not fair or safe, especially at this stage. This line also sounds like a #humblebrag. Cut. 

"This topic is not simple, and to understand how to make the best decisions for your personal situation, you have to slough through a lot of detail. But we're here to guide you through, and remember that you can ask questions along the way."